src/main/java/edu/nicholaidudakiwwarrick/advancedjava/StockQuoteApplication.java

```diff
@@ -0,0 +1,40 @@
+package edu.nicholaidudakiwwarrick.advancedjava;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormatter;
+
+import java.util.List;
+
+/**
+ * Basic class with a Main method to create StockService instances
+ */
+public class StockQuoteApplication {
+    /**
+     * @param args an array that contains elements:
+     *             <code>String</code> representation of stock symbol, start date, and end date
+     */
+    public static void main(String[] args) {
+
+        // create a StockService instance using the StockServiceFactory static factory method
+        StockService basicStockService = StockServiceFactory.newInstance();
```

Comment:
don't call it basicStockService - this client code has no idea about the concrete class. It is just dealing with the interface which is StockService.